SurveyTests/Sources/Specs/Domain/Use Case/Survey/SubmitSurveyUseCaseSpec.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SubmitSurveyUseCaseSpec.swift
+//  Survey
+//
+//  Created by Khanh on 27/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import Nimble
+import Quick
+@testable import Survey
+
+final class SubmitSurveyUseCaseSpec: QuickSpec {
+
+    override func spec() {
+        var submitSurveyUseCase: SubmitSurveyUseCase!
+        var surveyRepository: SurveyRepositoryMock!
+
+        describe("a SubmitSurveyUseCase") {
+
+            beforeEach {
+                surveyRepository = SurveyRepositoryMock()
+                submitSurveyUseCase = SubmitSurveyUseCase(surveyRepository: surveyRepository)
+            }
+
+            context("when submit is called") {
```

Comment:
```suggestion
            context("its submit") {
```

Reply:
@blyscuit No worry, I just fixed it at https://github.com/nkhanh44/ic-survey-ios/pull/84/commits/c7f6cf16549454272505562033fd062dee7e0b5c, thank you 🙏 .